extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -276,29 +282,56 @@ export class InstrumentationService {
   }
 
   /**
+   * @param {!Element} el
+   * @private
+   * @return {!Object<string, string>}
+   */
+  extractAnalyticsVariables_(el) {
```

Comment:
Can this be merged with [`dom. getDataParamsFromAttributes`](https://github.com/ampproject/amphtml/blob/master/src/dom.js#L405-L418)